[2025-09-08T17:09:39.721+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:09:39.733+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-04-20T12:00:00+00:00 [queued]>
[2025-09-08T17:09:39.740+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-04-20T12:00:00+00:00 [queued]>
[2025-09-08T17:09:39.740+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:09:39.756+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2025-04-20 12:00:00+00:00
[2025-09-08T17:09:39.760+0000] {standard_task_runner.py:72} INFO - Started process 20788 to run task
[2025-09-08T17:09:39.764+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskBranch', 'backfill__2025-04-20T12:00:00+00:00', '--job-id', '159', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpy8chs9qx']
[2025-09-08T17:09:39.766+0000] {standard_task_runner.py:105} INFO - Job 159: Subtask taskBranch
[2025-09-08T17:09:39.803+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskBranch backfill__2025-04-20T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:09:39.861+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-04-20T12:00:00+00:00'
[2025-09-08T17:09:39.863+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:09:39.892+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-09-08T17:09:39.900+0000] {python.py:240} INFO - Done. Returned value was: taskC
[2025-09-08T17:09:39.900+0000] {branch.py:38} INFO - Branch into taskC
[2025-09-08T17:09:39.900+0000] {skipmixin.py:230} INFO - Following branch {'taskC'}
[2025-09-08T17:09:39.909+0000] {skipmixin.py:278} INFO - Skipping tasks [('taskE', -1)]
[2025-09-08T17:09:39.929+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:09:39.929+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskBranch, run_id=backfill__2025-04-20T12:00:00+00:00, execution_date=20250420T120000, start_date=20250908T170939, end_date=20250908T170939
[2025-09-08T17:09:39.979+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:09:40.010+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:09:40.014+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
